gsoc2015/student_guidelines.md

```diff
@@ -0,0 +1,67 @@
+increase your chances, this guide covers the most important things
+to OpenAstronomy, however, you should also checkout the [GSoC Student
+Guide], the [Python Software Foundation] guide and the [SymPy
+Guidelines] to name but a few. 
+
+Some general pointers to follow are:
+
+
+1. **The better we know you, the better we can judge you.**
+There are many ways to get yourself known by the community, you can introduce yourself through the 
+mailing list, chat with us in the IRC channel and even contact the mentors to know
+more about certain project. As well as that, when you have particular idea in 
+mind, get in touch with that project and introduce yourself.
+
+2. **Be a user!**
+Download one or more of the OpenAstronomy projects and start experimenting with 
+the code. Look at the latest pictures of the million degree Sun with SunPy or 
+volume render a simulation of the galaxy with yt, there are loads of things to
+do with all the projects involved with OpenAstronomy, get involed!
+
+3. **Set yourself up as a developer.**
+Create yourself an account in [GitHub](http://github.com) or [BitBucket](http://bitbucket.org).
+Don't know how to use [git](http://www.git-scm.com/) or [Mercurial](http://mercurial.selenic.com/)?
+Don't worry, it's easy and there are lots of git [tutorials](http://try.github.com) [online](http://gitimmersion.com/) 
+and many [Mercurial](http://hginit.com/) [ones](http://hgbook.red-bean.com/read/)
+[aswell](http://mercurial.selenic.com/) that will help you to get quite confident with it in a short time.
+Also both the [Astropy](http://docs.astropy.org/en/stable/index.html#developer-documentation)
+and [yt](http://yt-project.org/docs/dev/developing/developing.html)
+developers guides are great reads no matter what project you are thinking of working on.
+
+4. **Start to be a developer.**
+Checkout the issue lists of the projects to see if there is something that you could do.
+You will be expected to submit a PR to one of the OpenAstronomy projects before
+you submit your application (it does not have to be accepted).
+
+5. **Plan your application.**
+Think which is your favourite project from the [ideas page](/gsoc2015/ideas.html) or think
+of a new one that will help out one of the OpenAstronomy members.
+Prepare a plan on how you will tackle that project and the time it will take you to solve it.
+Do not worry much in providing exact dates, but plan it as best as you can 
+(if you are planing to have some holidays during the GSoC programme, then you should mention it too).
+Also, don't be afraid to ask your potential mentor for help.
+Don't be shy when describing yourself!
+
+6. **Submit your application.**
+Don't forget to [submit your application](http://www.google-melange.com) before the deadline.  
+We are also happy to give comments on your proposal if you like.  Feel free to 
+add your application to the wiki following this format: "GSoC-2015 Application \<Your Name\>:\<Project Name\>"
```

Comment:
Should we ask to use a format like:
"GSoC-2015 Application \<sub-org\> \<Your Name\>:\<Project Name\>"